kastenwesen.py

```diff
@@ -225,11 +232,12 @@ def run(self, container_instance):
 
 
 class AbstractContainer(object):
-    def __init__(self, name, sleep_before_test=0.5, only_build=False):
+    def __init__(self, name, sleep_before_test=0.5, only_build=False, startup_gracetime=STARTUP_GRACETIME):
```

Comment:
I think this will render STARTUP_GRACETIMEs that are set in kastenwesen_config.py ineffective, since in Python mutable arguments in the default-values of methods are iirc evaluated at time of definition and kastenwesen_config.py is called after defining the method.
If I am correct the better way would be:
```python
def __init__(self, name, sleep_before_test=0.5, only_build=False, startup_gracetime=None):
   if startup_gracetime is None:
      self.startup_gracetime = STARTUP_GRACETIME
   else:
      self.startup_gracetime = startup_gracetime
```